rotkehlchen/tests/unit/decoders/test_convex.py

```diff
@@ -0,0 +1,995 @@
+                    hexstring_to_bytes('0x000000000000000000000000cf50b810e57ac33b91dcf525c6ddd9881b139332'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000008014595f2ab54cd7c604b00e9fb932176fdc86ae'),  # noqa: E501
+                ],
+            ), EthereumTxReceiptLog(
+                log_index=218,
+                data=hexstring_to_bytes('0x000000000000000000000000000000000000000000000001b8e10e82689017e0'),  # noqa: E501
+                address=string_to_ethereum_address('0x62B9c7356A2Dc64a1969e19C23e4f579F9810Aa7'),
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000000000000000000000000000000000000000000000'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000cf50b810e57ac33b91dcf525c6ddd9881b139332'),  # noqa: E501
+                ],
+            ), EthereumTxReceiptLog(
+                log_index=219,
+                data=hexstring_to_bytes('0x000000000000000000000000000000000000000000000001b8e10e82689017e0'),  # noqa: E501
+                address=string_to_ethereum_address('0x62B9c7356A2Dc64a1969e19C23e4f579F9810Aa7'),
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000cf50b810e57ac33b91dcf525c6ddd9881b139332'),  # noqa: E501
+                    hexstring_to_bytes('0x00000000000000000000000095c5582d781d507a084c9e5f885c77babacf8eea'),  # noqa: E501
+                ],
+            ), EthereumTxReceiptLog(
+                log_index=220,
+                data=hexstring_to_bytes('0x000000000000000000000000000000000000000000000001b8e10e82689017e0'),  # noqa: E501
+                address=CVX_REWARDS,
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0xe2403640ba68fed3a2f88b7557551d1993f84b99bb10ff833f0cf8db0c5e0486'),  # noqa: E501
+                    hexstring_to_bytes('0x00000000000000000000000095c5582d781d507a084c9e5f885c77babacf8eea'),  # noqa: E501
+                ],
+            ),
+        ],
+    )
+    dbethtx = DBEthTx(database)
+    dbethtx.add_ethereum_transactions([transaction], relevant_address=None)
+    decoder = EVMTransactionDecoder(
+        database=database,
+        ethereum_manager=ethereum_manager,
+        eth_transactions=eth_transactions,
+        msg_aggregator=msg_aggregator,
+    )
+    events = decoder.decode_transaction(transaction=transaction, tx_receipt=receipt)
+    expected_events = [
+        HistoryBaseEntry(
+            event_identifier='0xdaead2f96859462b5800584ecdcf30f2b83a1ca2c36c49a838b23e43c61d803f',
```

Comment:
```suggestion
            event_identifier=event_identifier,
```

Reply:
Same in other places

Reply:
Do you mean `event_identifier=tx_hex` ?

Reply:
yes, that should be. You have a variable defined before. Maybe I looked it wrong or mixed it at the time of writting